tests_func/load/mongo_load.go

```diff
@@ -0,0 +1,247 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/tinsane/tracelog"
+	testUtils "github.com/wal-g/wal-g/tests_func/utils"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"io/ioutil"
+	"sync"
+)
+
+var contextByName = map[string]context.Context{
+	"todo":       context.TODO(),
+	"background": context.Background(),
+}
+
+type MongoWorker struct {
+	wg sync.WaitGroup
+	ch chan interface{}
+	fs []func()
+}
+
+func CreateMongoWorker(chSz int) MongoWorker {
+	return MongoWorker{ch: make(chan interface{}, chSz)}
+}
+
+func (mw *MongoWorker) AddMongoOp(client *mongo.Client, sop string) error {
+	var arr []json.RawMessage
+	var err error
+	err = json.Unmarshal([]byte(sop), &arr)
+	if err != nil {
+		return err
+	}
+	for _, ropdata := range arr {
+		var opdata map[string]json.RawMessage
+		err = json.Unmarshal(ropdata, &opdata)
+		if err != nil {
+			return fmt.Errorf("command expected to be a map, but found %s", testUtils.GetStringType(opdata))
+		}
+		switch string(opdata["op"]) {
+		case `"c"`:
```

Comment:
Are ""-quotes necessary here?

Reply:
Yes, because in patrons command has "op": "c" and because of using RawMessage we need the case like this